src/components/OpenFileDialog/VisTypeFilter.js

```diff
@@ -0,0 +1,37 @@
+import i18n from '@dhis2/d2-i18n'
+import { Divider, SingleSelect, SingleSelectOption, colors } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { visTypeDisplayNames } from '../../modules/visTypes'
+import { VisTypeIcon } from '../VisTypeIcon'
+import { CustomSelectOption } from './CustomSelectOption'
+
+export const VisTypeFilter = ({ selected, onChange }) => (
+    <SingleSelect
+        selected={selected}
+        onChange={({ selected }) => onChange(selected)}
+        prefix={i18n.t('Type')}
+        dense
+    >
+        <SingleSelectOption label={i18n.t('All types')} value="all" />
+        <SingleSelectOption label={i18n.t('All charts')} value="chart" />
```

Comment:
I see that the spec says 

> The first two items, labelled All types and All charts, are separated with a divider.

but aren't we using `visualizations` nowdays?

Reply:
@cooper-joe ?

Reply:
PT is also a visualization type, but here we want to distinguish between charts and PT.

Reply:
So `Single Value` is a "chart"? 🤔

Reply:
Good point.
I think this denomination didn't change since the old favorites dialog was introduced, which happened before SV.
I'm not really sure, maybe these 2 filters don't make much sense anymore?
Perhaps we should have a more dynamic filter where you can check/uncheck the types you want to see in the list...
or, "All charts" should exclude both PT and SV (easy fix).

Reply:
@cooper-joe

Reply:
Added a TODO note.